gitlab/resource_gitlab_user.go

```diff
@@ -2,8 +2,10 @@ package gitlab
 
 import (
 	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
```

Comment:
please reorder

Reply:
done. we should consider golangci-lint since it can check for organized imports. #361